pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -347,9 +346,46 @@ type fileDescription struct {
 	OpenFlag uint32
 }
 
+func (fd *fileDescription) inode() *Inode {
+	return fd.vfsfd.Dentry().Impl().(*kernfs.Dentry).Inode().(*Inode)
```

Comment:
I think we can split this function like this:

```golang
func (fd *fileDescription) dentry() *kernfs.Dentry {
	return fd.vfsfd.Dentry().Impl().(*kernfs.Dentry)
}
func (fd *fileDescription) inode() *Inode {
	return fd.dentry().Inode().(*Inode)
}
```

But I'm not sure it's the best choice